includes/compat/class-wc-stripe-sepa-subs-compat.php

```diff
@@ -74,6 +76,52 @@ public function is_subs_change_payment() {
 		return ( isset( $_GET['pay_for_order'] ) && isset( $_GET['change_payment_method'] ) );
 	}
 
+	/**
```

Comment:
A large chunk of code duplicated. Can it be DRY and moved to a utility class or common ancestor, if there is one?